src/test/group/__snapshots__/remove-group-modal.test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<RemoveGroupModal /> should render correctly 1`] = `
```

Comment:
This snapshot is not very useful, use `shallow(...).dive()` (render one more layer of the component) or mount (to render everything)

Reply:
agree - removed this test